pkg/core/namespace.go

```diff
@@ -32,6 +32,7 @@ import (
 )
 
 const serviceAccountName = "riff-build"
+const persistentVolumeClaimName = "riff-cnb-cache"
```

Comment:
This value is entirely determined by riff-buildpack-group and not the riff cli.

Reply:
It's indirectly defined by the riff cli, in the [manifest#Namespace](https://github.com/projectriff/riff/blob/10dee37c4e5769b069cc1edeb861b8dcb24000a0/main.go#L50) section. I agree this is definitely a shortcut though but I'm not sure about the alternatives.

Shall we ask again for the manifest? I think it's not a viable option, because:

 - users could forget what manifest they used (you said so for secrets ;-))
 - users may have upgraded riff and the manifest may have changed completely

Shall we attach an annotation somewhere?

Reply:
If we slapped a label on all resource created by `namespace init` then we can find which resources should be deleted.

I agree that we don't want to require a manifest here. Ideally, the manifest should only be used for `system install` and never asked for again.

Reply:
Done in https://github.com/projectriff/riff/commit/e7d65db24ce39375ca3f3eb535ffb1ad4177c58d